libraries/AP_HAL_Linux/RCOutput_AioPRU.cpp

```diff
@@ -53,8 +61,16 @@ void RCOutput_AioPRU::init()
    // Reset PRU
    *ctrl = 0;
 
-   // Load firmware
+   // Loading firmware
+   // for ARM64 using loading PRU code in block instead memcpy
```

Comment:
Why?

Can we do this iteration on both arm32 and arm64 instead?

Reply:
Hi @peterbarker 
After a long discussion and tests in the beagleboard discord channel, the only way to use loading the pru code using shared memory was replacing the memcpy to loading the code in blocks. Trying to use the memcpy happen core dumps and you can see debuging the PRU the amount of code loaded was random. There are several theories, insights but the only way to use shared memory is changing from memcpy do blocks - the other way is using the remoteproc - what is I believe out of discussion because is purely filesystem process.